lib/plucky/criteria_hash.rb

```diff
@@ -164,14 +166,12 @@ def modifier_merge(hash, value)
 
     # Private
     def merge_values_into_array(value, other_value)
-      array(value).concat(array(other_value)).uniq
+      array(value).concat(array(other_value)).flatten.uniq
```

Comment:
I haven't unwound how this is used, but would this break docs storing nested array values?